code/modules/mob/living/carbon/human/life/handle_shock.dm

```diff
@@ -7,25 +7,25 @@
 	if(health < config.health_threshold_softcrit)
 		shock_stage = max(shock_stage, 60)//If they took too much damage, they immediately enter shock.
 
-	switch(traumatic_shock)
+	switch(max(0,traumatic_shock))
 		if(200 to INFINITY)
-			shock_stage = min(200,max(150,shock_stage++)) //Indescribable pain. At this point, you will immediately be knocked down, with shock stage set to 150.
+			shock_stage = CLAMP(shock_stage++,150,200) //Indescribable pain. At this point, you will immediately be knocked down, with shock stage set to 150.
 			return
 
 		if(199 to 100)
 			shock_stage = min(200,shock_stage++) //If their shock exceeds 100, add more to their shock stage, regardless of health.
 			return
 
 		if(99 to 75)
-			shock_stage = max(0, min(shock_stage, 120)) //No greater than 120 and no smaller than 0
+			shock_stage = CLAMP(shock_stage, 0, 120)) //No greater than 120 and no smaller than 0
 			return
 
 		if(74 to 50)
-			shock_stage = max(0, min(shock_stage - 2, 80))
+			shock_stage = CLAMP(shock_stage - 2, 0, 80))
 			return
 
 		if(49 to 0)
-			shock_stage = max(0, min(shock_stage - 10, 60)) //When we have almost no pain remaining. No greater than 60 and no smaller than 0, reduced by 10 each time.
+			shock_stage = CLAMP(shock_stage - 10, 0, 60)) //When we have almost no pain remaining. No greater than 60 and no smaller than 0, reduced by 10 each time.
```

Comment:
Why reducing the `shock_stage` so fast when we may still have `traumatic_shock`?
Go a little softer on the recovery.
This will definitely require some playtesting.

Reply:
Wait what?

49-0 is literally the traumatic shock they have, which is quite little.